include/fiction/algorithms/simulation/sidb/random_sidb_layout_generator.hpp

```diff
@@ -87,27 +87,35 @@ struct generate_random_sidb_layout_params
  * The layout skeleton serves as the starting layout to which SiDBs are added to create the final layout.
  *
  * @tparam Lyt SiDB cell-level SiDB layout type.
- * @param lyt_skeleton A layout to which random cells are added to create the final layout.
  * @param params The parameters for generating the random layout.
- * @return A randomly-generated layout of SiDBs.
+ * @param lyt_skeleton Optional layout to which random cells are added to create the final layout.
```

Comment:
```suggestion
 * @param skeleton Optional layout to which random cells are added to create the final layout.
```

Reply:
An uninformed reader might have trouble understanding the concept of a skeleton given this docstring. Can we expand it?